corehq/ex-submodules/phonelog/migrations/0005_add_forceclose_entry_20160408_1530.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phonelog', '0004_merge'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ForceCloseEntry',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('domain', models.CharField(max_length=100, db_index=True)),
```

Comment:
FYI I think this is going to create an unwanted `varchar_pattern_ops` index. You can either update the migration to add the index after the column is created or add an operation to delete the index (the name should be standard across all DB's)